scripts/resurrect-repositories.rb

```diff
@@ -55,7 +56,13 @@ def find_git_remote_url(git_cmd, remote_name)
 end
 
 def find_git_repos_from_disk(path)
-  Dir.glob("#{path}/**/.git").map { |d| d.sub('/.git', '') }.compact.sort.uniq
+  stderr = Tempfile.new()
```

Comment:
```suggestion
  begin
    paths = `find '#{path}' -name .git -type d -exec dirname {} \\; 2>#{stderr.path}`
    unless File.zero?(stderr)
      puts "WARNING: Following errors occurred when traversing directories for git repositories:".yellow
      puts `cat #{stderr.path}`.yellow
    end
    return paths.split("\n").sort
  ensure
    stderr.close
    stderr.unlink
  end
```

Reply:
wrapping within a `begin..ensure` block can be used to ensure that the file handle is also closed.

Reply:
Pushed changes with `begin..ensure`